### ISSUE TITLE
Roadmap for WP Job Manager (+ extensions) request / queries

### ISSUE THREAD

**Issue by Hastibe** [Automattic/WP-Job-Manager#2723]
Is there a roadmap for WP Job Manager development (and extensions, ideally)? After development of WP Job Manager having been very slow over the past several years (though all work has been appreciated!), it feels like more substantial development has been occurring recently--this is really exciting, and I was wondering (1) if it will continue, (2) where it is going, and (3) how we can provide input (in a more organized fashion that simply opening isolated enhancement requests or posting on the support forum).

In general, as a heavily-committed user and fan of this plugin since 2019, I'd love to understand more about who and what is driving the more substantial development of the plugin (and extensions) that seems to be happening recently, what the roadmap for future development is, and how users of the plugin can be more substantially involved in helping guide the future development of the plugin.

**Comment by thedebian** [comment]
Hi @Hastibe,

Thanks for the kind words.

We're currently focused on making WP Job Manager better by default, trying to sort out the most requested features, and think about improvements that could make the lives of everyone easier.
We do not have a formal roadmap for that, but we'll probably write a couple of blog posts to explain what's been done and what the next steps are.

For your third point "how we can provide input", support forums and Github issues are the right places, if you have a list of feedback you'd like to discuss, support forums are there for that too.

For further dicussions, please review the [support policy for WPJM](https://wpjobmanager.com/support-policy/) and use the right channel for your request, by posting to [the community forum](https://wordpress.org/support/plugin/wp-job-manager/), Github being there to report issues or make feature requests.

**Comment by Hastibe** [comment]
@thedebian -- you're welcome, and been meaning to reply to you for forever! I think it would be great for the plugin support and developers to work with plugin users to develop a formal roadmap. I do have lists of various features and bugs (that I've provided in the past), but I think the most important aspect to future WP Job Manager development for a lot of us is that it is done in coordination with @tripflex's plugins, which have been and continue to be indispensable.

> For further dicussions, please review the [support policy for WPJM](https://wpjobmanager.com/support-policy/) and use the right channel for your request, by posting to [the community forum](https://wordpress.org/support/plugin/wp-job-manager/), Github being there to report issues or make feature requests.

I'd welcome further roadmap-related discussions on the community forum; in fact, before I posted here, I asked about a roadmap for WP Job Manager [here on the community forum](https://wordpress.org/support/topic/2020-roadmap-add-your-2020-wishes-for-wp-job-manager/), but it didn't receive responses from anyone providing support for or contributions to the plugin.
